velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -51,9 +60,24 @@ class ReferenceQueryRunner {
     VELOX_UNSUPPORTED();
   }
 
+  /// Similar to above but for join node with 'probeInput' and 'buildInput'.
+  virtual std::vector<RowVectorPtr> executeVector(
+      const std::string& sql,
+      const std::vector<RowVectorPtr>& probeInput,
+      const std::vector<RowVectorPtr>& buildInput,
+      const RowTypePtr& resultType) {
+    VELOX_UNSUPPORTED();
+  }
+
   virtual std::vector<velox::RowVectorPtr> execute(const std::string& sql) {
     VELOX_UNSUPPORTED();
   }
+
+  virtual std::string createTable(
+      const std::vector<velox::RowVectorPtr>& input,
+      const std::string& tableName) {
+    VELOX_UNSUPPORTED();
+  }
```

Comment:
I don't think we should expose this as an API

For DuckQueryRunner, it's using an internal call
https://github.com/facebookincubator/velox/blob/ff07cb0b757050c2a631ede72f7f6cb0586c4d47/velox/exec/tests/utils/QueryAssertions.cpp#L893-L895

Reply:
This can be a private method if don't have an external use case? Thanks!

Reply:
Also chat offline with @kewang1024 , can you update the comment say this create an empty table with given data type and table name. The function returns the root directory of table files.